GitExtUtils/GitUI/DpiUtil.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
```

Comment:
Why has this moved from GitUI? This is UI specific...

Reply:
On my phone. IIRC it was too make it available to a plugin.

Reply:
ah, makes sense